plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -7,9 +7,9 @@
  * @package webp-uploads
  */
 
-use PerformanceLab\Tests\TestCase\ImagesTestCase;
+use WebP_Uploads\Tests\TestCase;
```

Comment:
Wouldn't this be less redundant?

```suggestion
use WebP_Uploads\Tests;
```

Reply:
For example, `Dominant_Color_Images\Tests` is being used.

Reply:
Did you mean to add a namespace? With such a namespace alias, tests are breaking.

Reply:
Oh sorry, I misread this.